### ISSUE TITLE
Browser exception when deployed under Nuxt 3

### ISSUE THREAD

**Issue by jcalfee** [andreyvit/json-diff#122]
argv is undefined
(has-flag.js:30:28)
![image](https://github.com/andreyvit/json-diff/assets/204121/3ea40efa-f625-44c4-8e6b-d9e2bf082ae0)
![image](https://github.com/andreyvit/json-diff/assets/204121/f27d576f-658b-4c75-a86d-cb877ad2202e)

Are you checking somewhere to see if this is node or the browser?

**Comment by AnthonyRuelle** [comment]
Same problem here,

This package is dead ? not maintained @andreyvit ?

**Comment by ewoudenberg** [comment]
@AnthonyRuelle  @jcalfee 

Sorry to see this issue. Note that `json-diff` is maintained, but the error appears to be in `colors` package.

**Comment by jcalfee** [comment]
True, but I'm not sure the `colors` would support the browser.  Chromium does not show colors in the console as far as I know.  I see now, you have an option `{ color: false }` already, I think I need to just use that.  I'm not setup to test now but that should do the trick.  Thanks

**Comment by AnthonyRuelle** [comment]
Please reopen this issue.
With color: false the problem persist :/

**Comment by jcalfee** [comment]
In that case, a dynamic import of the `colors` (if enabled) may be the way to go.

**Comment by AnthonyRuelle** [comment]
even setting color: false, the processing crashes

![image](https://github.com/andreyvit/json-diff/assets/93582464/d3371bd6-d268-47e9-8f64-56f94cb31e25)

Has anyone managed to resolve this issue?

**Comment by soarding** [comment]
Same problem here, Has anyone managed to resolve this issue
